ui/shared/utils/constants.js

```diff
@@ -55,6 +55,17 @@ export const PROJECT_FIELDS = [
   GENOME_VERSION_FIELD,
 ]
 
+export const BASE_UPLOAD_FORMATS = [
```

Comment:
variable names that make sense scoped within a file often don't make sense when they are exported. Base of what? This should be renamed `FILE_FORMATS`

Reply:
okay